org.jacoco.examples.test/pom.xml

```diff
@@ -38,6 +38,11 @@
       <groupId>${project.groupId}</groupId>
       <artifactId>org.jacoco.agent.rt</artifactId>
     </dependency>
+    <dependency>
```

Comment:
with recent version of Maven this can be build before the maven plugin, so need to be sure the maven plugin is build first.